notebooks/03-query/20-tiering-historicals.ipynb

```diff
@@ -0,0 +1,790 @@
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea52acbb-826c-4022-ab8a-5d79fc2eda0a",
+   "metadata": {},
+   "source": [
+    "Consider that, because of [time partitioning](https://druid.apache.org/docs/latest/multi-stage-query/concepts#partitioning-by-time), some parts of this query were calculated on the `slow` historical tier, and some were calculated on historicals in the `_default_tier` tier."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37e887eb-f2b9-4014-b2bd-e2e0186a56f3",
+   "metadata": {},
+   "source": [
+    "### Load data according to age\n",
+    "\n",
+    "Run the cell below to ingest some data for the current year.\n",
+    "\n",
+    "The TIME_EXTRACT function is used to fake data for this year by calculating the shift between the source timestamp and today's date."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3ccb4e5-0845-4985-a427-1dc1381afa7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "INSERT INTO \"''' + table_name + '''\"\n",
+    "WITH \"ext\" AS (\n",
+    "  SELECT *\n",
+    "  FROM TABLE(\n",
+    "    EXTERN(\n",
+    "      '{\"type\":\"http\",\"uris\":[\"https://druid.apache.org/data/wikipedia.json.gz\"]}',\n",
+    "      '{\"type\":\"json\"}'\n",
+    "    )\n",
+    "  ) EXTEND (\"isRobot\" VARCHAR, \"channel\" VARCHAR, \"timestamp\" VARCHAR, \"flags\" VARCHAR, \"isUnpatrolled\" VARCHAR, \"page\" VARCHAR, \"diffUrl\" VARCHAR, \"added\" BIGINT, \"comment\" VARCHAR, \"commentLength\" BIGINT, \"isNew\" VARCHAR, \"isMinor\" VARCHAR, \"delta\" BIGINT, \"isAnonymous\" VARCHAR, \"user\" VARCHAR, \"deltaBucket\" BIGINT, \"deleted\" BIGINT, \"namespace\" VARCHAR, \"cityName\" VARCHAR, \"countryName\" VARCHAR, \"regionIsoCode\" VARCHAR, \"metroCode\" BIGINT, \"countryIsoCode\" VARCHAR, \"regionName\" VARCHAR)\n",
+    ")\n",
+    "SELECT\n",
+    "  TIME_SHIFT(TIME_PARSE(\"timestamp\"), 'P1Y', (TIME_EXTRACT(CURRENT_TIMESTAMP,'YEAR') - TIME_EXTRACT(TIME_PARSE(\"timestamp\"),'YEAR'))) AS \"__time\",\n",
+    "  \"isRobot\",\n",
+    "  \"channel\",\n",
+    "  \"isUnpatrolled\",\n",
+    "  \"page\",\n",
+    "  \"comment\",\n",
+    "  \"commentLength\",\n",
+    "  \"user\"\n",
+    "FROM \"ext\"\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "\n",
+    "display.run_task(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f976d910-d4e9-43d6-ad8c-4f03bc4dd762",
+   "metadata": {},
+   "source": [
+    "Review the following retention load rules configuration, then run the cell to apply it to the table.\n",
+    "\n",
+    "- `loadByPeriod`, giving a [period load rule](https://druid.apache.org/docs/latest/operations/rule-configuration/#period-load-rule) covering data newer than one year old, requesting one cached replica on both the `_default_tier` and `slow` tiers.\n",
+    "- `loadForever`, requesting all data to be cached on the `slow` tier."
```

Comment:
```suggestion
    "The following retention load rules configuration creates load rules as follows:\n",
    "\n",
    "- `loadByPeriod`, creates a [period load rule](https://druid.apache.org/docs/latest/operations/rule-configuration/#period-load-rule) for one cached replica of data newer than one year on both the `_default_tier` and `slow` tiers.\n",
    "- `loadForever`, creates a rule to cache all remaining data on the `slow` tier."
```
i think we should be careful about "load forever" requesting "all data" on the slow tier. It's really only applying to the data that was not handled by the first load rule, right?